zendesk/organization_membership.go

```diff
@@ -0,0 +1,65 @@
+package zendesk
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+)
+
+type (
+	// OrganizationMembership is struct for organization membership payload
+	// https://developer.zendesk.com/api-reference/ticketing/organizations/organization_memberships/
+	OrganizationMembership struct {
+		ID             int64     `json:"id,omitempty"`
+		URL            string    `json:"url,omitempty"`
+		UserID         int64     `json:"user_id"`
+		OrganizationID int64     `json:"organization_id"`
+		Default        bool      `json:"default"`
+		Name           string    `json:"organization_name"`
+		CreatedAt      time.Time `json:"created_at,omitempty"`
+		UpdatedAt      time.Time `json:"updated_at,omitempty"`
+	}
+
+	// OrganizationMembershipListOptions is a struct for options for organization membership list
+	// ref: https://developer.zendesk.com/api-reference/ticketing/organizations/organization_memberships/
+	OrganizationMembershipListOptions struct {
+		PageOptions
+		OrganizationID int64 `json:"organization_id,omitempty" url:"organization_id,omitempty"`
+		UserID         int64 `json:"user_id,omitempty" url:"user_id,omitempty"`
+	}
+
+	// OrganizationMembershipAPI is an interface containing organization membership related methods
+	OrganizationMembershipAPI interface {
```

Comment:
Would you add this to https://github.com/nukosuke/go-zendesk/blob/bc077cbdabe5e6f1e6d3e66687cca0fc94776f24/zendesk/api.go#L7-L31
and regenerate mock by `go generate ./...`?